lib/mapper.js

```diff
@@ -113,7 +113,7 @@ function formatConversionTag(msg) {
   var ret = {
     rdid: device.advertisingId,
     idtype: getIdType(device),
-    lat: device.adTrackingEnabled ? 1 : 0,
+    lat: device.adTrackingEnabled ? 0 : 1,
```

Comment:
The name of the parameter can make this sentence a bit confusing to read, but we're checking if the tracking is enabled with this param, not the LIMITING of the tracking. Double-check in case I'm wrong, please! :)